packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -491,7 +491,7 @@
     "@Lists": {},
     "no_prodcut_in_list": "There is no product in this list",
     "no_product_in_section": "There is no product in this section",
-    "recently_seen_products": "My recently seen products",
+    "recently_seen_products": "History",
```

Comment:
Can you maybe add a comment for the translators

Reply:
Thank you @M123-dev for your review!
I have no idea how translators work and I don't know which details they need: how short is the label supposed to be? on what kind of Widget is it displayed? what is the context?
In a first approach with a tag called "recently_seen_products" and "History" as a translation in English they have 90% of the data they need, I guess.

Reply:
"This goes to a header, so please keep it short."  or something like that

Reply:
That would make sense. Actually so far there are only two labels in `app_en.arb` with a "keep it short" hint...